src/main/java/org/jabref/gui/groups/GroupDialogViewModel.java

```diff
@@ -263,6 +263,24 @@ private void setupValidation() {
         });
     }
 
+    /**
+     * gets the absolute path relative to getLatexFileDirectory, if given a relative path
+     * @param input the user input path
+     * @return an absolute path
+     */
+    private Path getAbsTexGroupPath(String input) {
+        Path texFilePath;
+        // check whether LatexFileDirectory is set
+        if (currentDatabase.getMetaData().getLatexFileDirectory(preferencesService.getUser()).isPresent()) {
+            Path latexFileDirectory = currentDatabase.getMetaData().getLatexFileDirectory(preferencesService.getUser()).get();
+            // a relative filepath should be relative to latexFileDirectory by default
+            texFilePath = latexFileDirectory.resolve(input);
+        } else {
+            texFilePath = Path.of(input);
+        }
+        return texFilePath;
```

Comment:
So this is nitpicking, include a suggestion if you believe it improves readability, otherwise don't.

* I'd suggest using an `Optional<Path>` variable for the latex file directory. It is easier to see what is going on when you are using `latexFileDirectory.` `isPresent`/`get` rather than `currentDatabase.getMetaData().getLatexFileDirectory(preferencesService.getUser()`.
* While opinions differ on the readability of multiple return statements, I'd suggest using two in this situation rather than creating the extra variable `texFilePath`. The return statements would be right next to each other and personally, I'd find it more readable with two return statements in this situation.
* You can take advantage of [Optional.map](https://docs.oracle.com/en/java/javase/15/docs/api/java.base/java/util/Optional.html#map(java.util.function.Function)) combined with `.orElse` to make this method very short, but perhaps it lacks in readability.
    ```
    Optional<Path> latexFileDirectory = currentDatabase.getMetaData().getLatexFileDirectory(preferencesService.getUser());
    return latexFileDirectory.map(path -> path.resolve(input)).orElse(Path.of(input));
    ```